lib/galaxy/webapps/galaxy/api/datatypes.py

```diff
@@ -112,6 +116,15 @@ async def converters(self) -> DatatypeConverterList:
         """Gets the list of all installed converters."""
         return view_converters(self.datatypes_registry)
 
+    @router.get(
+        '/api/datatypes/suitable_converters',
```

Comment:
This isn't consuming the collection ID I don't think. You'll probably want an API test to make sure the old and new API endpoints both work.

Reply:
I moved the endpoint to dataset_collections, which doesn't use FastAPI yet, it seems.

I'll work on a test, all the same.  There are a few endpoints I created in the process of working on this ticket, should each one have an api test?